### PR TITLE
[202012] [Mellanox] Update SDK\FW ver. 4.4.3216\2008.3216

### PR DESCRIPTION
**Pull request by dprital**
Signed-off-by: Dror Prital <drorp@nvidia.com>

#### Why I did it

**Changes and new features:**

* Added support in SN4600C systems for new module Finisar ET7402-CWDM4 (100G CWDM4 QSFP28 1310nm SM 2KM).
* Added support for new module MMS1W50-HM (2km transceiver FR4) for 200GbE
* Improved performance of "per-port-buffer" counters
* Added support for Kernel 5.10

**Bug fix:**

* On rare occasions (0.5%), in SN4600C systems, when using 100GbE NRZ mode and Fastboot flow, the link up time may take up to 10 seconds

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [X] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: platform/mellanox/fw.mk [hunk 1]
```diff
@@ -11,17 +11,17 @@ else
 FW_FROM_URL = n
 endif
 
-MLNX_SPC_FW_VERSION = 13.2008.3110
+MLNX_SPC_FW_VERSION = 13.2008.32180
```
Comment (liat-grozovik) [inline platform/mellanox/fw.mk:14]: wrong FW version. please fix

**Review by liat-grozovik — CHANGES_REQUESTED** [review]

File: platform/mellanox/fw.mk (hunk 1 above)
Comment (dprital) [reply platform/mellanox/fw.mk]: Fixed. Thanks!

**Review by liat-grozovik — APPROVED** [review]

**Comment by abdosi** [comment]
@liat-grozovik and @dprital  these PR is conflicting for 201911. Please create one for 201911.

cc @prsunny

**Comment by prsunny** [comment]
@dprital , @liat-grozovik , could you please raise the PR today, we would need to start the build asap.

**Comment by dgsudharsan** [comment]
@prsunny The pull request is raised here. Can you please reivew? https://github.com/Azure/sonic-buildimage/pull/8145

**Pull request merged**